core/templates/components/button-directives/exploration-embed-button-modal.component.html

```diff
@@ -22,14 +22,14 @@ <h3>Embed this exploration</h3>
       To embed this exploration in another webpage, copy the following HTML into the source code of the webpage:
     </p>
 
-    <div class="oppia-embed-modal-code" ng-click="selectText($event)">&lt;iframe src="<[serverName]>/embed/exploration/<[explorationId]>" width="700" height="1000"&gt;</div>
+    <div class="oppia-embed-modal-code" (click)="selectText($event)">&lt;iframe src="{{serverName}}/embed/exploration/{{explorationId}}" width="700" height="1000"&gt;</div>
```

Comment:
Not possible, angular2+ string interpolation is `{{ ... }}`

Reply:
cc @seanlip Since Jinja is removed should we start using `{{` and `}}` in all the Angular 8+ files?

Reply:
Can we (safely) do it universally instead of just the Angular 8+ files? I think using it in just a few files is going to cause confusion -- this sort of thing should be made to the whole codebase at once and announced very clearly to the developers.

Reply:
@seanlip That's one way to look at this. The other viewpoint would be that interpolation is the only thing that is present in almost all, if not all, directives. And since Mariana is changing directives to angularjs components and renaming the files as well, having different interpolations would really help us distinguish ajs component templates from angular components templates, especially during code review. Since some changes need to made in HTML files that are not the template of the file that is being migrated. In such cases, the migration syntax is different based on whether it is an ajs template or angular template.

Reply:
"Interpolation ... is present in almost all, if not all, directives" is a strong statement that needs proof. Without this, you can't rely on it.

If there is a desire to distinguish migrated from non-migrated files, then rather than relying on directives which may or may not exist, I suggest changing the naming convention so that Angular 2+ files are named using whatever our eventual naming convention is, and AngularJS files are named using some suffix that includes `ajs`. That would be more foolproof.

Reply:
Hmm... well then I will make a pr that changes ajs interpolation from `<[ ]>` to `{{ }}`.
BTW, the count was 288/394. The 394 contains files like mainpage.html, header-css-libs.html, no-script, etc. When checked for `.directive.html` and `.component.html`, it was 247/262.

Reply:
OK, sounds good, thanks. If you are planning to have different interpolation conventions then please also add a lint check to ensure that we don't have the wrong interpolation in the wrong type of file.